app/src/main/res/layout/organiser_post_event.xml

```diff
@@ -1,211 +1,217 @@
                 android:layout_height="wrap_content"
-                android:text="Entrant Limit (optional)"
                 android:layout_marginTop="12dp"
                 android:fontFamily="@font/roboto_mono_medium"
-                android:textSize="18sp"
-                android:textColor="@color/dark_purple"/>
+                android:text="Entrant Limit (optional)"
+                android:textColor="@color/dark_purple"
+                android:textSize="18sp" />
 
             <EditText
                 android:id="@+id/o_postEvent_limit_edit"
                 android:layout_width="match_parent"
                 android:layout_height="60dp"
                 android:ems="10"
-                android:textSize="18sp"
-                android:inputType="number"
+                android:fontFamily="@font/roboto_mono_medium"
                 android:hint="Event Limit"
+                android:inputType="number"
                 android:textColor="@color/dark_purple"
-                android:fontFamily="@font/roboto_mono_medium"/>
+                android:textSize="18sp" />
 
             <TextView
                 android:id="@+id/o_postEvent_winners"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:text="Number of Winners"
                 android:layout_marginTop="12dp"
                 android:fontFamily="@font/roboto_mono_medium"
-                android:textSize="18sp"
-                android:textColor="@color/dark_purple"/>
+                android:text="Number of Winners"
+                android:textColor="@color/dark_purple"
+                android:textSize="18sp" />
 
             <EditText
                 android:id="@+id/o_postEvent_winners_edit"
                 android:layout_width="match_parent"
                 android:layout_height="60dp"
                 android:ems="10"
-                android:textSize="18sp"
-                android:inputType="number"
+                android:fontFamily="@font/roboto_mono_medium"
                 android:hint="Event Number of Winners"
+                android:inputType="number"
                 android:textColor="@color/dark_purple"
-                android:fontFamily="@font/roboto_mono_medium"/>
+                android:textSize="18sp" />
 
             <CheckBox
                 android:id="@+id/o_postEvent_geolocation_check"
                 android:layout_width="match_parent"
                 android:layout_height="60dp"
                 android:layout_marginTop="12dp"
                 android:checked="false"
-                android:textColor="@color/dark_purple"
-                android:textSize="18sp"
                 android:fontFamily="@font/roboto_mono_medium"
-                android:text="Enable Geolocation" />
+                android:text="Enable Geolocation"
+                android:textColor="@color/dark_purple"
+                android:textSize="18sp" />
```

Comment:
are these edits just to make them all in the same order?

Reply:
I dont even think i changed these, when I was playing around in the visual layout editor I think that's when it reordered these.

Reply:
linting perhaps